nightfall-client/src/services/state-sync.mjs

```diff
@@ -36,6 +36,20 @@ export const syncState = async (
   console.log('From block', fromBlock);
   const stateContractInstance = await waitForContract(STATE_CONTRACT_NAME); // Rollback, BlockProposed
 
+  // console.log(
```

Comment:
Remove this code

Reply:
same, this is marked outdated by github, as I removed this code after your suggestion